tests/devapps/Managed Identity apps/ManagedIdentityAppVM/Program.cs

```diff
@@ -20,6 +21,7 @@
     try
     {
         var result = await mi.AcquireTokenForManagedIdentity(scope)
+            .WithMtlsProofOfPossession()
```

Comment:
Should you also add the new WithExtraQueryParams()?

Reply:
not needed I think